compile_proto.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Just remember to adjust it after we decide the final form in the elixir server SDK